### ISSUE TITLE
PHP 8.1 Support - PHP 7.4 EOL

### ISSUE THREAD

**Issue by steven-hoffman-jomashop** [YotpoLtd/magento2-module-yotpo-reviews#293]
Hi,

As you are aware PHP 7.4 is End of Life on 28 Nov 2022. (https://www.php.net/supported-versions.php).
This module 's composer.json does not support PHP 8.1 (https://github.com/YotpoLtd/magento2-module-yotpo-reviews/blob/master/composer.json#L10)

Does this module compatible with PHP 8.1?
If yes, can you update the composer.json and release a new version?
If no, can you add compatibility PHP 8.1

**Comment by steven-hoffman-jomashop** [comment]
Someone has opened a PR for this: https://github.com/YotpoLtd/magento2-module-yotpo-reviews/pull/291

**Comment by LiorGingi** [comment]
Hi @steven-hoffman-jomashop,

On Mar. 8th we've released [version 3.3.0](https://github.com/YotpoLtd/magento2-module-yotpo-reviews/releases/tag/3.3.0) which is fully compatible with PHP 8.1.
This version is not merged to Master due to various reasons.

The new version is already available in Packagist and can be installed via composer;
```
composer require yotpo/module-review:3.3.0 --no-update
composer update 'yotpo/*'
php bin/magento maintenance:enable
php bin/magento setup:upgrade
php bin/magento setup:di:compile
php bin/magento setup:static-content:deploy
php bin/magento maintenance:disable
php bin/magento cache:clean
```

Thanks,
Lior Gingihashvili.
